### DISCUSSION TITLE
ait-wiki/development/utils/

### DISCUSSION THREAD

**Discussion by giscus** [amblelabs/ait-wiki#8] [General]
# ait-wiki/development/utils/

AIT provides and uses a lot of utility classes and methods. Here are some of them.
Core UtilsCore utilities are common side utility classes which can be used on both client and server.
Scheduler Allows you to schedule tasks. Tasks do not persist between restarts.
ForcedChunkUntil Allows you to continuously force load a chunk.
ServerLifecycleHooks Allows you to get the MinecraftServer instance.
ShapeUtil Allows you to rotate a VoxelShape.
StackUtil Allows you to spawn items in a world, save items and item stacks to NBT, play break sounds & more.

https://pavatus.github.io/ait-wiki/development/utils/

**Comment by duzos** [comment]
safe*

**Comment by duzos** [comment]
sens => send
